app/assets/stylesheets/layout/_drop_down.sass

```diff
@@ -49,7 +49,6 @@
 .dropdown .dropdown-panel,
 .toolbar .legacy-actions-more
   min-width: 200px
-  max-width: 360px
```

Comment:
Hey @HDinger , can you foresee any issues with the removal of this restriction? The label `Open fullscreen view` would otherwise be caught by the ellipsis and I haven't noticed any differences in other toolbars. I would rather prefer we do not cut off labels in context menus.